src/Controller/ListPageRssController.php

```diff
@@ -164,12 +192,68 @@ public function build(NodeInterface $node): CacheableResponse {
    *
    * @param \Drupal\node\NodeInterface $node
    *   The node containing the list.
+   * @param \Drupal\Core\Cache\CacheableMetadata $cache_metadata
+   *   The current cache metadata.
    *
    * @return array
    *   Array of items to be rendered.
    */
-  protected function getItemList(NodeInterface $node): array {
-    return [];
+  protected function getItemList(NodeInterface $node, CacheableMetadata $cache_metadata): array {
+    $configuration = ListPageConfiguration::fromEntity($node);
+    // Always sort by the updated date descending.
+    $configuration->setSort([
+      'name' => 'changed',
+      'direction' => 'DESC',
+    ]);
+    $execution_result = $this->listExecutionManager->executeList($configuration);
+    $query = $execution_result->getQuery();
+    $results = $execution_result->getResults();
+    $cache_metadata->addCacheableDependency($query);
+    $cache_metadata->addCacheTags(['search_api_list:' . $query->getIndex()->id()]);
+    $result_items = [];
+    foreach ($results->getResultItems() as $item) {
+      /** @var \Drupal\Core\Entity\ContentEntityInterface $entity */
+      $entity = $item->getOriginalObject()->getEntity();
+      $cache_metadata->addCacheableDependency($entity);
+      $description = '';
+      // Add a default description if the body is available.
+      if ($entity->hasField('body')) {
+        $description = $entity->get('body')->view([
+          'type' => 'text_default',
+          'label' => 'hidden',
+          'settings' => [],
+        ]);
+        // We need to escape the results of renderPlain in order to maintain
```

Comment:
Is that what we are doing? p tags are not lost in twig. Twig autoesapes but leaves basic markup..